oops/hosts/juno/sru/__init__.py

```
@@ -0,0 +1,277 @@
+import oops
+
+from oops.hosts.juno import Juno
+
+from filecache import FCPath
+
+################################################################################
+# Standard class methods
+################################################################################
+
+#===============================================================================
+def from_file(filespec, return_all_planets=False, method='strict', **parameters):
```

Ruff (0.16.1)

[warning] Boolean default positional argument in function definition

(FBT002)